src/constants/constant.ts

```diff
@@ -1,15 +1,36 @@
-const BASE_URL = 'https://api.themoviedb.org/3';
+const API = {
+  URL: 'https://api.themoviedb.org/3/',
+  LANGUAGE: 'ko-KR',
+};
+
+const PATH = {
+  POPULAR_MOVIE: 'movie/popular',
+  SEARCHED_MOVIE: 'search/movie',
+  DETAIL_MOVIE_INFO: 'movie',
+};
 
-const ENDPOINT = {
-  POPULAR_MOVIES: '/movie/popular',
-  MOVIE_SEARCH: '/search/movie',
+const RATING_MESSAGES: Record<string, string> = {
+  '0': '별점 미등록',
+  '2': '최악이에요',
+  '4': '별로예요',
+  '6': '보통이에요',
+  '8': '재밌어요',
+  '10': '명작이에요',
 };
```

Comment:
as const 를 사용하면 직접 타입 선언을 안해줘도 될 것 같아요 :) 

```suggestion
const RATING_MESSAGES= {
  '0': '별점 미등록',
  '2': '최악이에요',
  '4': '별로예요',
  '6': '보통이에요',
  '8': '재밌어요',
  '10': '명작이에요',
} as const;
```

Reply:
as const로 변경했더니
`Element implicitly has an 'any' type because expression of type 'string' can't be used to index type`
요런 에러가 뜨네요😭😭

Reply:
아하. as const 를 사용하면 타입이 더 타이트하게 잡혀서 그럴거예요.
저는 이럴 때 해당 사용처에 타입단언을 추가해주는 편이긴해요.

```ts
const RATING = 0 | 2 | 4 | 6 | 8 | 10;

// 아마 이런 코드에서 에러가 났겟죠?
const rating = document.querySelector('').value as RATING;
RATING_MESSAGES[rating];
```
이런식으로 별점에 올 수 있는 숫자 타입을 따로 분리해서 타입단언시에 사용해도 좋겠네요 ㅎㅎ